install_deps.sh

```diff
@@ -363,6 +363,19 @@ git clone -b v0.7.0 --depth 1 https://github.com/jaegertracing/jaeger-client-cpp
     cd ${HOME} && \
     rm -r jaeger-client-cpp
 
+# Cppcheck 2.8
```

Comment:
apt-get install?

Reply:
apt-get install will install cppCheck v 1.4 which is very old and doesn't even support c++17.
The only way to install the newest version is via ppa or building from the official download site sourceforge.
I havn't found a ppa.